infra/base-images/base-builder/Dockerfile

```diff
@@ -139,10 +136,8 @@ RUN cd $SRC && \
     rm -rf examples $SRC/oss-fuzz.tar.gz
 
 # Do precompiles before copying other scripts for better cache efficiency.
-COPY precompile_afl /usr/local/bin/
```

Comment:
I think I have these on seperate lines on purpose. So that caching works better when I'm working on these scripts.

Reply:
I see. Let me revert this change.